org.eu.wabarc.wayback.yml

```diff
@@ -0,0 +1,34 @@
+app-id: org.eu.wabarc.wayback
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: wayback
+finish-args:
+  - --share=network
+  - --filesystem=xdg-documents
+modules:
+  - name: wayback
+    buildsystem: simple
+    build-commands:
+      - |
+        set -eu pipefail
+        tar -xvf wayback-linux-*.tar.gz
+        mv wayback-linux-* wayback
+      - install -Dm755 -t /app/bin wayback
```

Comment:
there is an inconsistency in the path here. line 16 `wayback` doesn't exist, but `mv` expect a directory.
line 17 `wayback` is expected to be a file